### PR TITLE
docs: add alejandrocuba to GDE resources

### PR DESCRIPTION
**Pull request by alejandrocuba**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by MarkTechson** [comment]
Generating a preview so we can move this forward

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -45,6 +45,14 @@
     "picture": "alan-agius4.jpg",
     "bio": "Angular CLI Member, Loves TypeScript, Build Tools, Food, Beer & Coffee :)"
   },
+  "alejandrocuba": {
+    "name": "Alejandro Cuba Ruiz",
+    "picture": "alejandrocuba.jpg",
```
Comment (AndrewKushnir) [inline aio/content/marketing/contributors.json:50]: It looks like the mentioned image was not added to `aio/content/images/bios` folder. Could you please upload the image that should be used on angular.io?

Comment (alejandrocuba) [reply aio/content/marketing/contributors.json:50]: @AndrewKushnir thanks for sharing the path to the image directory. PR updated.

**Review by MarkTechson — APPROVED** [review]
LGTM, Welcome.

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit faf3b45d481c8b28d4807c80c9e127b18e8a9140.

**Pull request closed without merging**